### ISSUE TITLE
undefined method `zone' for Time:Class

### ISSUE THREAD

**Issue by map7** [sgruhier/capistrano-db-tasks#106]
I'm getting this error with the setup suggested in capistrano-db-tasks;

```
NoMethodError: undefined method `zone' for Time:Class
/home/map7/code/capistrano-db-tasks-master/lib/capistrano-db-tasks/database.rb:44:in `c
urrent_time'                                                                          
/home/map7/code/capistrano-db-tasks-master/lib/capistrano-db-tasks/database.rb:48:in `o
utput_file'                                                                           
/home/map7/code/capistrano-db-tasks-master/lib/capistrano-db-tasks/database.rb:147:in `
db_dump_file_path'                                                                    
/home/map7/code/capistrano-db-tasks-master/lib/capistrano-db-tasks/database.rb:130:in `
clean_dump_if_needed'                                                                 
/home/map7/code/capistrano-db-tasks-master/lib/capistrano-db-tasks/database.rb:214:in `
ensure in remote_to_local'                                                            
/home/map7/code/capistrano-db-tasks-master/lib/capistrano-db-tasks/database.rb:214:in `
remote_to_local'                                                                      
/home/map7/code/capistrano-db-tasks-master/lib/capistrano-db-tasks/dbtasks.rb:41:in `bl
ock (4 levels) in <top (required)>'                                                   
/usr/local/rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/sshkit-1.11.4/lib/sshkit/backe
nds/abstract.rb:29:in `instance_exec'                                                 
/usr/local/rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/sshkit-1.11.4/lib/sshkit/backe
nds/abstract.rb:29:in `run'                                                           
/usr/local/rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/sshkit-1.11.4/lib/sshkit/runne
rs/parallel.rb:12:in `block (2 levels) in execute'                                    
NoMethodError: undefined method `zone' for Time:Class
/home/map7/code/capistrano-db-tasks-master/lib/capistrano-db-tasks/database.rb:44:in `c
urrent_time'                                                                          
/home/map7/code/capistrano-db-tasks-master/lib/capistrano-db-tasks/database.rb:48:in `o
utput_file'                                                                           
/home/map7/code/capistrano-db-tasks-master/lib/capistrano-db-tasks/database.rb:147:in `
db_dump_file_path'                                                                    
/home/map7/code/capistrano-db-tasks-master/lib/capistrano-db-tasks/database.rb:120:in `
dump'                                                                                 
/home/map7/code/capistrano-db-tasks-master/lib/capistrano-db-tasks/database.rb:212:in `
remote_to_local'                                                                      
/home/map7/code/capistrano-db-tasks-master/lib/capistrano-db-tasks/dbtasks.rb:41:in `bl
ock (4 levels) in <top (required)>'                                                   
/usr/local/rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/sshkit-1.11.4/lib/sshkit/backe
nds/abstract.rb:29:in `instance_exec'                                                 
/usr/local/rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/sshkit-1.11.4/lib/sshkit/backe
nds/abstract.rb:29:in `run'     
/usr/local/rbenv/versions/2.2.5/lib/ruby/gems/2.2.0/gems/sshkit-1.11.4/lib/sshkit/runne
rs/parallel.rb:12:in `block (2 levels) in execute'                                    
Tasks: TOP => db:pull => db:local:sync
```

I had to add the following to my deploy.rb.
`require "active_support/all"`

This should be added to the readme or something as without this line it cannot find 'zone'.

**Comment by numbata** [comment]
@map7 thanks for the report. Can you try new version? Is it resolve this issues in your case?

**Comment by map7** [comment]
Yes this issue is fixed in 0.6
